File: src/api/FunCoding.ReVision/FunCoding.ReVision.WebApi/Models/ReviseRequest.cs
namespace FunCoding.ReVision.WebApi.Models;

public class ReviseRequest
{
    public string Draft { get; set; } = string.Empty;
    public string TargetLanguage { get; set; } = string.Empty;
    public string UserLanguage { get; set; } = string.Empty;
    public string WritingTone { get; set; } = string.Empty;
}

File: src/outlook-add-in/ReVision/src/taskpane/models/reviseRequest.ts
import { Suggestion } from "./suggestResponse";

export class ReviseRequest {
  draft: string;
  targetLanguage: string;
  writingTone: string;
  suggestions: Suggestion[];

  constructor(draft: string, targetLanguage: string, writingTone: string, suggestions: Suggestion[]) {
    this.draft = draft;
    this.targetLanguage = targetLanguage;
    this.writingTone = writingTone;
    this.suggestions = suggestions;
  }
}
